File: Python/pass_or_fail.py
s1 = int(input("Enter your marks in Subject 1\n"))
s2 = int(input("Enter your marks in Subject 2\n"))
s3 = int(input("Enter your marks in Subject 3\n"))

sum = s1+s2+s3

if((sum/3)>40 and s1>33 and s2>33 and s3>33):
    print("You're Pass")
else:
    print("You're fail")

File: C Programming/Chapter 3/Pass_or_fail.c
#include <stdio.h>

int main()
{
    int English, Mathematics, Science;
    float Total;

    printf("Enter your Marks in English\n");
    scanf("%d", &English);

    printf("Enter your Marks in Mathematics\n");
    scanf("%d", &Mathematics);

    printf("Enter your Marks in Science\n");
    scanf("%d", &Science);

    Total=(English+Mathematics+Science)/3;


    if ((Total<40) || English<16.5 || Mathematics<16.5 || Science<16.5)

    {
        printf("Your Total Percentage is %f and You are Fail\n", Total);
        
    }
    else
    {
        printf("Your Total Percentage is %f and You are Pass\n", Total);
    }

    return 0;
}
